src/app/client/src/app/modules/profile/components/profile-badge/profile-badge.component.html

```diff
@@ -17,7 +17,23 @@
           </div>
         </div>
       </div>
-      <div *ngIf="badge.createdTS || badge.createdTs" class="meta">{{badge.createdTS || badge.createdTs | dateFormat}}</div>
+      <div class="meta">{{badge.createdDate || badge.createdTs | dateFormat: 'MMM YYYY'}}</div>
```

Comment:
Fix not applied on latest change in angular-migration branch.

Expected:
<div *ngIf="badge.createdTS || badge.createdTs" class="meta">{{badge.createdTS || badge.createdTs | dateFormat: 'MMM YYYY'}}</div>